src/dune_rules/cram/cram_exec.ml

```diff
@@ -60,8 +60,7 @@ end = struct
     in
     let open Fiber.O in
     let+ () =
-      Process.run ~display:!Clflags.display ~stdin_from ~stdout_to Strict prog
-        argv
+      Process.run ~display:Quiet ~stdin_from ~stdout_to Strict prog argv
```

Comment:
When a cram test runs a process, we don't want the actual program inside being reported to the user.